modules/system/console/OctoberUtil.php

```diff
@@ -294,8 +295,47 @@ protected function utilPurgeUploads()
         if (!$this->confirmToProceed('This will PERMANENTLY DELETE files in the uploads directory that do not exist in the "system_files" table.')) {
             return;
         }
-
-        // @todo
+        
+        $totalCount = 0;
+        $uploadsPath = Config::get('filesystems.disks.local.root', storage_path('app'));
+        $uploadsPath .= '/uploads';
+        
+        /*
+         * Recursive function to scan the directory for files and ensure they exist in system_files.
+         */
+        $purgeFunc = function ($targetDir) use (&$purgeFunc, &$totalCount) {
+            if ($files = File::glob($targetDir.'/*')) {
+                foreach ($files as $file) {
+                    if (!is_file($file)) {
+                        continue;
+                    }
+                    
+                    $model = FileModel::where('disk_name', basename($file));
+                    if ($model->count()) {
+                        continue;
+                    }
```

Comment:
No need to have `$model` as a separate variable here, and besides, it's not a Model, it's a QueryBuilder. Please just call the `->count()` directly on the where and put it all in the if statement

Reply:
Updated!